lib/matplotlib/widgets.py

```diff
@@ -188,24 +193,29 @@ def __init__(self, ax, label, image=None,
         self.color = color
         self.hovercolor = hovercolor
 
+    @cbook.deprecated("3.4")
+    @property
+    def cnt(self):
+        # Not real, but close enough.
```

Comment:
`Widget.cnt` is roughly the next cid to be returned, but this is just the count of the callbacks added. I can't get the next cid without reaching into `CallbackRegistry` internals (and maybe not even then since it uses `itertools.count()`.)